esx_service/cli/vmdkops_admin.py

```diff
@@ -591,7 +591,8 @@ def get_service_status():
         if output[2] == "not":
             return NOT_RUNNING_STATUS
 
-        pid = output[3].split("=")[1]
```

Comment:
please add a comment with expected result example (makes easy to understand and check the code).